docs/index.md

```diff
@@ -2,15 +2,29 @@
 title: <b>Wash</b>
 ---
 
-<p class="intro-normal">A UNIX-like shell for managing your cloud native and non-cloud native things. <a href="{{ '/getting_started' | relative_url }}"><strong>Get started today</strong></a></p>
+<div class="flex-row">
+  <p class="intro-normal">A UNIX-like shell for managing your cloud native and non-cloud native things.</p>
+  <div class="flex-column">
+    <a class="intro-normal center btn" href="{{ '/getting_started' | relative_url }}">GET&nbsp;STARTED</a>
+    <a class="intro-normal center underline version">Version</a>
+  </div>
+</div>
+<script type="text/javascript">
+$.get("https://api.github.com/repos/puppetlabs/wash/releases/latest", function(data) {
+  $('.version')
+    .text("Version " + data.tag_name)
+    .attr("href", "https://github.com/puppetlabs/wash/releases/tag/" + data.tag_name)
+    .attr("title", "Released " + (new Date(data.published_at).toLocaleDateString()))
```

Comment:
Why include the version? Thinking it's enough to include the date, i.e. something like `Last release: <date>` to indicate project activity. Also, why (visibly) link to the repo here? Thinking in terms of minimizing distractions for the user (only link they should think of clicking in home page, if any, is "Getting Started" -- I don't see a good reason for them to click the repo link). Maybe that's a non-issue though.

Reply:
Also, `Getting Started` button feels like a bit of an eye-sore. Have you gotten feedback from other people about it?

Reply:
I liked the design of https://www.rust-lang.org.

Reply:
Linking to the repo is useful for people who are returning and see there's a new version.

Reply:
Also, I disagree about it being an eyesore. Although I think someone more experience could probably make it look better.

Reply:
Ok, making it easy for returning people to see there's a new release is a good reason. Instead of linking via version, could we link via date? Thinking

"Last release: today"; "Last release: 3 days ago"; "Last release: 1 week ago" (until certain time period where we just list the date)

That would also (cleanly) indicate project activity.


Re: eyesore bit, I'm OK with what you have if other people think it looks good b/c that's definitely subjective.

Reply:
I think version is more recognizable. I included release date as a hover popup (title). I think "Last release: ..." would take up more space than I want in the design.